client/lib/paths/test/index.js

```diff
@@ -1,64 +1,50 @@
 /** @format */
-
 /**
  * External dependencies
  */
-
 import { expect } from 'chai';
 
 /**
  * Internal dependencies
  */
-import paths from '../';
```

Comment:
sidebar: for internal libs I don't mind `import * as paths`. Especially for test files that don't get bundled.